mm/2s2h/Rando/ActorBehavior/EnStoneheishi.cpp

```diff
@@ -0,0 +1,38 @@
+#include "ActorBehavior.h"
+#include <libultraship/libultraship.h>
+
+extern "C" {
+#include "variables.h"
+#include "overlays/actors/ovl_En_Stone_Heishi/z_en_stone_heishi.h"
```

Comment:
Linux doesn't like the casing here, which might be my fault. The "Heishi" should be "heishi":

`#include "overlays/actors/ovl_En_Stone_heishi/z_en_stone_heishi.h"`